src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -161,12 +161,39 @@ public WPI_TalonSRX getLeftMaster() {
     public WPI_TalonSRX getRightMaster() {
         return rightMaster;
     }
+        /**
+     * Gets the left follower talon
+     *
+     * @return The leftFollower talon
+     *
+     * @since 1.0.0
+     */
+    public WPI_TalonSRX getLeftFollower() {
+        return leftFollower;
+        
+    }
+
+    /**
+     * Gets the right Folllower talon
+     *
+     * @return The right Follower talon
+     *
+     * @since 1.0.0
+     */
+    public WPI_TalonSRX getRightFollower() {
+        return rightFollower;
+    }
+
+    public double test(){
+        return 12.90;
+    }
```

Comment:
Temporary code? Is this needed or called upon anywhere? If not, please remove before merging.